cfgmgr/intfmgr.cpp

```diff
@@ -843,6 +857,30 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
                 m_loopbackIntfList.insert(alias);
                 SWSS_LOG_INFO("Added %s loopback interface", alias.c_str());
             }
+
+            if (adminStatus.empty())
+            {
+                adminStatus = "up";
+            }
+            else if (adminStatus != "up" && adminStatus != "down")
+            {
+                SWSS_LOG_ERROR("Got incorrect value for admin_status as %s for intf %s, defaulting as up", adminStatus.c_str(), alias.c_str());
+                adminStatus = "up";
+            }
+
+            try
+            {
+                if(setIntfAdminStatus(alias, adminStatus))
+                {
+                    FieldValueTuple newAdminFvTuple("admin_status", adminStatus);
```

Comment:
is this new attribute for Loopback handled in orchagent? I believe, it should be ok. but can you please check?

Reply:
admin_status handling is skipped in intforch for loopback:
https://github.com/sonic-net/sonic-swss/blob/master/orchagent/intfsorch.cpp#L877

The set operation upon admin_status update will basically return after ensuring that the interface itself exists in syncd, and not perform any change as such.